### PR TITLE
PHP CodeSniffer fixes

### PR DESCRIPTION
**Pull request by tersmitten**

### CODE REVIEW

File: View/ClearCache/files.ctp [hunk 1]
```diff
@@ -17,9 +17,9 @@
 <table class="debug-table"><?php
 	$mask = empty($this->params['pass']) ? '.*' : implode(' ', $this->params['pass']);
 	$headers = array(__d('clear_cache', 'File Name (%s)', $mask), __d('clear_cache', 'Result'));
-	if (empty($files)) {
```
Comment (ceeram) [inline View/ClearCache/files.ctp:20]: i prefer : and endif; in .ctp files

Comment (ceeram) [inline View/ClearCache/files.ctp:20]: ow, im stupid, thats what you did, totally misreading the diff

**Comment by tersmitten** [comment]
No problem.

**Comment by ceeram** [comment]
Forgot about this one. Thanks

**Pull request merged**